src/main/java/de/tum/cit/aet/artemis/assessment/web/ResultResource.java

```diff
@@ -343,9 +346,42 @@ public ResponseEntity<FeedbackAnalysisResponseDTO> getFeedbackDetailsPaged(@Path
      * @return A {@link ResponseEntity} containing the maximum count of feedback occurrences (long).
      */
     @GetMapping("exercises/{exerciseId}/feedback-details-max-count")
-    @EnforceAtLeastInstructorInExercise
+    @EnforceAtLeastEditorInExercise
     public ResponseEntity<Long> getMaxCount(@PathVariable long exerciseId) {
         long maxCount = resultService.getMaxCountForExercise(exerciseId);
         return ResponseEntity.ok(maxCount);
     }
+
+    /**
+     * GET /exercises/{exerciseId}/feedback-details-participation : Retrieves paginated details of students affected by specific feedback entries for a specified exercise.
+     * <br>
+     * This endpoint returns details of students whose submissions were impacted by specified feedback entries, including student information
+     * and participation details. This supports efficient loading of affected students' data by returning only the required information.
+     * <br>
+     * Supports pagination and sorting, allowing the client to control the amount and order of returned data:
+     * <ul>
+     * <li><b>Pagination:</b> Controls page number and page size for the response.</li>
+     * <li><b>Sorting:</b> Allows sorting by specified columns (e.g., "participationId") in ascending or descending order.</li>
+     * </ul>
+     *
+     * @param exerciseId  The ID of the exercise for which the participation data is requested.
+     * @param feedbackIds A list of feedback IDs to filter affected students by specific feedback entries.
+     * @param data        A {@link PageableSearchDTO} object containing pagination and sorting parameters:
+     *                        <ul>
+     *                        <li>Page number and page size</li>
+     *                        <li>Sorted column and sorting order</li>
+     *                        </ul>
+     * @return A {@link ResponseEntity} containing a {@link Page} of {@link FeedbackAffectedStudentDTO}, each representing a student affected by the feedback entries, including:
+     *         <ul>
+     *         <li>List of affected students with participation details.</li>
+     *         <li>Total number of students affected (for pagination).</li>
+     *         </ul>
+     */
+    @GetMapping("exercises/{exerciseId}/feedback-details-participation")
+    @EnforceAtLeastEditorInExercise
+    public ResponseEntity<Page<FeedbackAffectedStudentDTO>> getParticipationWithFeedback(@PathVariable long exerciseId, @RequestParam List<String> feedbackIds,
```

Comment:
Can you also add 1-2 tests for the new endpoint?

Reply:
ok i implemented one thing, client side test also checks for the correct feedback beeing returned. Server test checks for correct and proper reply